src/components/footer.tsx

```diff
@@ -25,7 +24,7 @@ const navigation = {
     },
     {
       name: 'Instagram',
-      href: '#',
+      href: 'https://www.instagram.com/outboulder/?hl=en',
```

Comment:
I think I'd remove the `hl=en` query param. The href value should just be `https://www.instagram.com/outboulder`